package.json

```diff
@@ -19,21 +19,22 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/chai": "^4.3.3",
-    "@types/chai-as-promised": "^7.1.5",
+    "@types/chai": "^4.3.11",
+    "@types/chai-as-promised": "^7.1.8",
     "@types/mocha": "^9.1.1",
-    "chai": "^4.3.6",
+    "chai": "^4.3.10",
     "chai-as-promised": "^7.1.1",
     "mocha": "^10.0.0",
     "nock": "^13.2.9",
     "nyc": "^15.1.0",
     "prettier": "^2.7.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@babel/traverse": "^7.23.2"
   },
   "dependencies": {
-    "axios": "^0.27.2",
+    "axios": "^1.6.7",
```

Comment:
thats not working because there are multiple major changes inside, please keep at 0.27

Reply:
As for security alert, https://github.com/eclipse-velocitas/velocitas-project-generator-npm/security/dependabot/3 we need to calculate effort to bulk version or calculate and mitigate related risks

Reply:
We had troubles with this new major axios version in the past.

Reply:
Will be tracked in https://github.com/eclipse-velocitas/velocitas-project-generator-npm/issues/34